.eslintrc.json

```diff
@@ -103,7 +103,7 @@
         "no-unused-labels": 1,
         "no-useless-call": 1,
         "no-useless-concat": 1,
-        "no-useless-escape": 1,
+        "no-useless-escape": 0,
```

Comment:
Maybe, doesn't look like this affects any changes here; so not sure what it is referring to. Might have been left over from when you had changed things in deprecation file?